my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
```

Comment:
It's generally good practice to follow PEP-8 conventions on the ordering of imports, which puts Imports should be grouped in the following order: standard library imports, related third party imports and then local application/library specific imports.